### ISSUE TITLE
NetOffice.WordApi.Document.Revisions.AcceptAll() does not accept footnote changes

### ISSUE THREAD

**Issue by jozefizso** [NetOfficeFw/sandbox-codeplex-issues#16]
**This issue was imported from [CodePlex](https://netoffice.codeplex.com/workitem/17339)**

**[tobs1701](https://www.codeplex.com/site/users/view/tobs1701)** wrote 2012-02-02 at 21:39
Accidentally discovered this while processing some Word Documents that were undergoing massive revision. The Revisions.AcceptAll() method does not actually accept any of the changes in Word footnotes.

 

By contrast, _Document.AcceptAllRevisions() method accepts everything.

 

Sample documents and code are attached.

**[tobs1701](https://www.codeplex.com/site/users/view/tobs1701)** wrote 2012-02-02 at 21:39
Forgot to mention: I'm using the NetOffice Api build 1.3.2. 

Thanks, 

Toby

**[SebastianDotNet](https://www.codeplex.com/site/users/view/SebastianDotNet)** wrote 2012-02-03 at 01:06
are you sure this is a NetOffice issue? 

in other words: working with interop assemblies works as well and NetOffice ignore the footnotes?

i do a test with the interop assemblies in vs2010 and word 2007 and i have exactly the same behavior.

**[tobs1701](https://www.codeplex.com/site/users/view/tobs1701)** wrote 2012-02-03 at 16:34
It didn't occur to me that it might be a fundamental issue with the Microsoft API, but it looks as if you're right.

Thanks for checking into it.
